packages/core/src/Engine.ts

```diff
@@ -622,7 +624,7 @@ export class Engine extends EventDispatcher {
     if (physics) {
       initializePromises.push(
         physics.initialize().then(() => {
-          PhysicsScene._nativePhysics = physics;
+          Engine._nativePhysics = physics;
```

Comment:
Maybe use `this.engine` is better!

Reply:
ColliderShape cannot get engine, use console.warn remind the user